job-service/src/main/java/com/hpe/caf/services/job/api/JobsPut.java

```diff
@@ -262,16 +251,41 @@ private static void closeQueueConnection(final QueueServices queueServices)
                 throw new RuntimeException("Unknown taskDataEncoding");
             }
         } else if (taskDataObj instanceof Map<?, ?>) {
-            try {
-                taskDataBytes = codec.serialise(taskDataObj);
-            } catch (CodecException e) {
-                throw new RuntimeException("Failed to serialise TaskData", e);
-            }
+            taskDataBytes = serializingData(taskDataObj);
         } else {
             throw new RuntimeException("The taskData is an unexpected type");
         }
 
         return taskDataBytes;
     }
-
+    
+    private static byte[] serializingData(final Object taskDataObj)
```

Comment:
You don't normally use the `-ing` form of the verb when naming methods.  Can this be changed to `serializeData`?

Reply:
Changed